readthedocs/source/orb/operationcaching.md

```diff
@@ -0,0 +1,84 @@
+# Operation/Resolution Caching
```

Comment:
Can the contents of this doc be added to the sidetree.md doc instead of creating a new one?

Reply:
This is not part of Sidetree spec but I can do that.